### ISSUE TITLE
[Accessibility] Missing alt attribute (Amazon Pay)

### ISSUE THREAD

**Issue by amberhinds** [woocommerce/woocommerce-gateway-amazon-pay#333]
## Audit Information
This issue came from an audit WooCommerce had Equalize Digital complete in December 2025. I'm moving the issue out of Linear (WooA11Y-418) to this repo.

- EQD Audit ID: WooExt-513
- Component/Page: Amazon pay modal
- Affected Element: Amazon Pay logo
- URL where issue was discovered: https://apay-woo.mystagingwebsite.com/cart/

## WCAG Compliance

- Success Criterion: 1.1.1 Non-text Content
- WCAG Version: 2
- Level: A
- Severity: Medium

## Issue Description

This image doesn't have alt text assigned, nor does it have an empty alt attribute (alt=""). Screen reader users will perceive this image as "Unlabelled," which is not descriptive and may be confusing for the users.

### Impacted Populations

Blind and low vision people, Screen reader users (blind/low vision and neurodivergent people)

## Recommended Fix

Ensure all images have an alt attribute that clearly describes the image content or purpose on the page. This image should have alternative text that matches the text in the image.

## Code Example

`<img src="https://m.media-amazon.com/images/G/01/APay/UX/CheckoutAssets/Sandbox/Pay_with_Sandbox_Logo._CB657256914_.png">`

## Screenshots

https://drive.google.com/file/d/1yVZezADQlydQZeg407_DSLeZUbwTXNsi/view?usp=drivesdk

**Comment by ChristianAF** [comment]
Amazon Pay Sites issues
